src/System.Runtime/tests/System/Attributes.netstandard1.7.cs

```diff
@@ -137,7 +137,11 @@ public static class GetCustomAttribute
         public static void customAttributeCount()
         {
             List<CustomAttributeData> customAttributes =  typeof(GetCustomAttribute).Module.CustomAttributes.ToList();
-            Assert.Equal(3, customAttributes.Count);
+            // [System.Security.UnverifiableCodeAttribute()]
+            // [TestAttributes.FooAttribute()]
+            // [TestAttributes.ComplicatedAttribute((Int32)1, Stuff = 2)]
+            // [System.Diagnostics.DebuggableAttribute((Boolean)True, (Boolean)False)]
```

Comment:
Nit: rather than having these comments, would it make sense to codify this, e.g. have asserts that each of these is in the list?